### PR TITLE
WRR-992: Added `pageIndicatorPosition` prop in PageViews component

### PR DESCRIPTION
**Pull request by juwonjeong**
Enact-DCO-1.0-Signed-off-by: Juwon Jeong (juwon.jeong@lge.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
According to the updated UX guide, pageIndicator position can be either 'top' and 'bottom' 

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added `pageIndicatorPosition` prop in PageViews component to allow changing pageIndicator position 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
As discussed with the review, I added `pageIndicatorPosition` prop privately for now, since the UX guide has not been officially released. 

### Links
[//]: # (Related issues, references)
WRR-992

### CODE REVIEW

**Review by 0x64 — APPROVED** [review]
🈴

File: PageViews/PageViews.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import handle, {forwardCustomWithPrevent} from '@enact/core/handle';
 import kind from '@enact/core/kind';
+import {cap} from '@enact/core/util';
 import EnactPropTypes from '@enact/core/internal/prop-types';
```
Comment (MikyungKim) [inline PageViews/PageViews.js:4]:
```suggestion
import EnactPropTypes from '@enact/core/internal/prop-types';
import {cap} from '@enact/core/util';
```

File: samples/sampler/stories/default/PageViews.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ PageViews.displayName = 'PageViews';
 
 const Config = mergeComponentMetadata('PageViews', PageViews);
 const propOptions = {
+	pageIndicatorPosition: ['top', 'bottom'],
```
Comment (MikyungKim) [inline samples/sampler/stories/default/PageViews.js:11]: Let's add this control when this prop became public.

File: tests/screenshot/apps/components/PageViews.js [hunk 3]
```diff
@@ -16,23 +16,39 @@ const PageComponents = [
...
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={2} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
 		wrapper: {full: true}
 	},
 	{
-		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews fullContents index={2}>{PageComponents}</PageViews></Panel>,
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews fullContents index={2} pageIndicatorPosition="top">{PageComponents}</PageViews></Panel>,
+		wrapper: {full: true}
+	},
+	{
+		component: <Panel><Header title="title of panel" subtitle="subtitle of panel" /><PageViews index={0} pageIndicatorPosition="bottom">{PageComponents}</PageViews></Panel>,
```
Comment (MikyungKim) [inline tests/screenshot/apps/components/PageViews.js:39]: If `pageIndicatorPosition` has the default value `bottom`, can't we omit the prop?

File: PageViews/PageViews.module.less [hunk 4]
```diff
@@ -55,8 +55,12 @@
...
 		position: absolute;
 		align-self: center;
-		bottom: @sand-pageviews-full-steps-position-bottom;
-		min-height: @sand-pageviews-full-steps-area-min-height;
+	}
+
+	&.fullContents.indicatorBottom {
+		.steps {
+			bottom: @sand-pageviews-fullCotents-indicator-bottom-steps-bottom;
+		}
```
Comment (MikyungKim) [inline PageViews/PageViews.module.less:63]: how about
```suggestion
	&.fullContents.indicatorBottom .steps {
		bottom: @sand-pageviews-full-contents-bottom-steps-bottom;
	}
```

File: styles/variables.less [hunk 5]
```diff
@@ -764,8 +764,7 @@
 @sand-pageviews-navigation-button-area-width: 216px;
 @sand-pageviews-steps-area-min-height: 132px;
 @sand-pageviews-dot-steps-area-min-height: @sand-pageviews-steps-area-min-height;
-@sand-pageviews-full-steps-area-min-height: @sand-pageviews-steps-area-min-height;
-@sand-pageviews-full-steps-position-bottom: 0;
+@sand-pageviews-fullCotents-indicator-bottom-steps-bottom: 0;
```
Comment (MikyungKim) [inline styles/variables.less:767]:
```suggestion
@sand-pageviews-full-contents-bottom-steps-bottom: 0;
```

File: PageViews/PageViews.module.less (hunk 4 above)
Comment (juwonjeong) [reply PageViews/PageViews.module.less:63]: It's fixed.

File: styles/variables.less (hunk 5 above)
Comment (juwonjeong) [reply styles/variables.less:767]: It's fixed.

File: samples/sampler/stories/default/PageViews.js (hunk 2 above)
Comment (juwonjeong) [reply samples/sampler/stories/default/PageViews.js:11]: It's removed.

File: PageViews/PageViews.js (hunk 1 above)
Comment (juwonjeong) [reply PageViews/PageViews.js:4]: It's fixed.

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**